act-rules-format.bs

```diff
@@ -52,66 +48,105 @@ The separation between atomic rules and composite rules creates a division of re
     <li>Header indicated by using the `headers` attribute</li>
     <li>Header indicated by using ARIA labels</li>
   </ul>
-  <p>If any one of these rules passes, the test result of the composite rule for the table cell is "passed".</p>
+  <p>If any one of these rules passes, the outcome of the composite rule for the table cell is "passed".</p>
 </div>
 
+The separation between atomic rules and composite rules creates a division of responsibility. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules test if a combination of outcomes from atomic rules satisfy the conformance for an accessibility requirement.
```

Comment:
This may be confusing as this separation of responsibility is not always the case, as stated in the following paragraph: sometimes an atomic rule itself would test for conformance to an entire accessibility requirement, sometimes it requires a composed rule. My suggestion is simply to drop this paragraph, or make it a "note" block with a slight reword like

> The separation (...) ~creates~ _can be seen as_ a division of responsibility. Atomic rules ~test~ _typically test_ if web content is correctly implemented in a particular solution, while composite rules ~test~ _would test_ if a combination of outcomes from atomic rules satisfy the conformance for an accessibility requirement ~.~ _as a whole_.”

Reply:
agree with the reworded suggestion.

Reply:
Discussed on the call. We'll leave it in place, not as a note but taking over the editorial changes